Document/Structure/ArticleBridge.php

```diff
@@ -30,14 +30,26 @@ class ArticleBridge extends StructureBridge implements RoutableStructureInterfac
 
     public function getView(): string
     {
-        return $this->structure->getView();
+        /** @var string $view */
+        $view = $this->structure->getView();
+
+        return $view;
     }
 
+    /**
+     * @return string
+     */
     public function getController()
     {
-        return $this->structure->getController();
+        /** @var string $controller */
+        $controller = $this->structure->getController();
+
+        return $controller;
```

Comment:
Same as above no additional variable required:

```suggestion
        /** @var string */
        return $this->structure->getController();
```

Reply:
that's cool, didn't know about that